data/mships.txt

```diff
@@ -4564,8 +4564,8 @@ ship "CWC-2 Celestine"
 	explode "small explosion" 30
 	explode "medium explosion" 20
 	"final explode" "final explosion small"
-	description `Custom built wine tanker for the Luna Winery for transporting wine in bulk and preserving them in the most perfect condition with individual thermal control system for each tank to keep the heat as stable as possible.`
-	description `Because these ships are lightly armed, they rarely wander away from relative safety of heavily patrolled Paradise Planets.`
+	description `A custom-built wine tanker for the Luna Winery made to transport wine in bulk and preserve it in the most perfect condition, with individual thermal control systems for each tank to keep the heat as stable as possible.`
+	description `	Because these ships are lightly armed, they rarely wander away from the relative safety of heavily patrolled Paradise Planets.`
```

Comment:
```suggestion
	description `A custom-built wine tanker for the Luna Winery made to transport wine in bulk and preserve it in the most perfect condition, with individual thermal control system for each tank to keep the heat as stable as possible.`
	description `	Because these ships are lightly armed, they rarely wander away from the relative safety of heavily patrolled Paradise Planets.`
```